### PR TITLE
Fix/cosmoskit ci

### PR DESCRIPTION
**Pull request by elix1er**
revert to v16, as v18 not required
might fix breaking @cosmoskit/station-extension CI + avoids it's dependents being forced into using node v18

### CODE REVIEW

**Comment by elix1er** [comment]
unnecessary routine commits adding node engine v18 enforcement in package.json should be good to merge, 
regarding github workflow ci builds & tests, all passing on both versions as expected, could even be adjusted to version matrix builds @ v16 +  v18 . 

But the package.json engine node being **enforced** to be >=18 breaks tons of libraries using feather.js, stopping @cosmology @cosmos-kit wallet connector monorepo from passing CI checks.

Many GitHub actions and dev environments using any sub nodejs version 18 atm are failing to build, CI, and also prevents NPM version releases due to the failing tests now. 

I kindly ask for a review @octalmage @simcheolhwan @hanjukim, it's not even a big deal here, editing that one line of code to NOT enforce v18 yet on ANYTHING that got feathers.js as dependency listed. The tests pass, LGTM ? 

Thanks if anyone would find some minutes reviewing this PR,
~with  best regards **eL1x**

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@terra-money/feather.js",
-  "version": "1.0.3",
+  "name": "@elix1er/feather.js",
```
Comment (elix1er) [inline package.json]: ok this must be changed OBVIOUSLY before merge.

Comment (elix1er) [reply package.json]: Solved

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terra-money/feather.js",
-  "version": "1.0.3",
+  "version": "1.0.4",
```
Comment (elix1er) [inline package.json:3]: i guess this would be needed for triggering new npmjs deploy right?

**Review by elix1er — COMMENTED** [review]
Looks fine for me, just check version bump in package.json @ review

**Comment by elix1er** [comment]
#22 https://github.com/cosmology-tech/cosmos-kit/issues/190

**Comment by alecande11** [comment]
replaced by #24

**Pull request closed without merging**